components/common-go/kubernetes/kubernetes.go

```diff
@@ -69,6 +69,9 @@ const (
 
 	// workspaceCpuBurstLimit denotes the cpu burst limit of a workspace
 	WorkspaceCpuBurstLimitAnnotation = "gitpod.io/cpuBurstLimit"
+
+	// workspaceNetConnLimit denotes the maximum number of connections a workspace can make per minute
+	WorkspaceNetConnLimitAnnotation = "gitpod.io/netConnLimit"
```

Comment:
I felt it would be good that the unit is included in the annotation name.
```suggestion
	WorkspaceNetConnLimitAnnotation = "gitpod.io/netConnLimitPerMin"
```